how_to_build.md

```diff
@@ -11,7 +11,7 @@ At the time of writing the build is only confirmed to work on Windows using the
 
 1. Ensure you have .NET framework 4.6.1 or later installed.
 
-1. Ensure you have Visual Studio 2017 or MSBuild 15 installed.
+1. Ensure you have Visual Studio 2017 or MSBuild 15.3 installed.

```

Comment:
OK, apparently it's also the version of MSBuild... I still find the wording confusing, though. I think it's supposed to mean "(Visual Studio 2017 or MSBuild) 15.3", but I read it as "(Visual Studio 2017) or (MSBuild 15.3)"